http-netty/src/main/java/io/micronaut/http/netty/NettyHttpHeaders.java

```diff
@@ -292,8 +295,48 @@ public OptionalLong contentLength() {
         return OptionalLong.empty();
     }
 
+    @Override
+    public List<MediaType> accept() {
+        // use HttpHeaderNames instead of HttpHeaders
+        return MediaType.orderedOf(getAll(HttpHeaderNames.ACCEPT));
+    }
+
+    @Nullable
+    @Override
+    public Charset acceptCharset() {
+        String text = get(HttpHeaderNames.ACCEPT_CHARSET);
+        if (text == null) {
+            return null;
+        }
+        text = HttpHeadersUtil.splitAcceptHeader(text);
+        if (text != null) {
+            try {
+                return Charset.forName(text);
+            } catch (Exception ignored) {
+            }
+        }
+        // default to UTF-8
+        return StandardCharsets.UTF_8;
+    }
+
+    @Nullable
+    @Override
+    public Locale acceptLanguage() {
```

Comment:
Same question is there some performance related reason for not just returning optional?